css/Style.css

```diff
@@ -0,0 +1,866 @@
+.pt-3 {
+  padding-top: 3rem;
+}
+
+.pb-3 {
+  padding-bottom: 3rem;
+}
+
+.pl-3 {
+  padding-left: 3rem;
+}
+
+.pr-3 {
+  padding-right: 3rem;
+}
+
+.m-4 {
+  margin: 4rem;
+}
+
+.p-4 {
+  padding: 4rem;
+}
+
+.mt-4 {
+  margin-top: 4rem;
+}
+
+.mb-4 {
+  margin-bottom: 4rem;
+}
+
+.ml-4 {
+  margin-left: 4rem;
+}
+
+.mr-4 {
+  margin-right: 4rem;
+}
+
+.pt-4 {
+  padding-top: 4rem;
+}
+
+.pb-4 {
+  padding-bottom: 4rem;
+}
+
+.pl-4 {
+  padding-left: 4rem;
+}
+
+.pr-4 {
+  padding-right: 4rem;
+}
+
+.m-5 {
+  margin: 5rem;
+}
+
+.p-5 {
+  padding: 5rem;
+}
+
+.mt-5 {
+  margin-top: 5rem;
+}
+
+.mb-5 {
+  margin-bottom: 5rem;
+}
+
+.ml-5 {
+  margin-left: 5rem;
+}
+
+.mr-5 {
+  margin-right: 5rem;
+}
+
+.pt-5 {
+  padding-top: 5rem;
+}
+
+.pb-5 {
+  padding-bottom: 5rem;
+}
+
+.pl-5 {
+  padding-left: 5rem;
+}
+
+.pr-5 {
+  padding-right: 5rem;
+}
+
+.border {
+  border: 2px solid lightgray;
+}
+
+.border-right {
+  border-right: 2px solid lightgray;
+}
+
+.border-left {
+  border-left: 2px solid lightgray;
+}
+
+.border-top {
+  border-top: 2px solid lightgray;
+}
+
+.border-bottom {
+  border-bottom: 2px solid lightgray;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.font-bold {
+  font-weight: bold;
+}
+
+.text-primary {
+  color: #ce973e;
+}
+
+.text-secondary {
+  color: rgba(74, 74, 74, 0.7);
+}
+
+.text-uppercase {
+  text-transform: uppercase;
+}
+
+.text-underline {
+  text-decoration: underline;
+}
+
+.display-1 {
+  font-size: 1.25rem;
+}
+
+.display-2 {
+  font-size: 2.5rem;
+}
+
+.display-3 {
+  font-size: 3.75rem;
+}
+
+.text-right {
+  text-align: right;
+}
+
+.button, .button-primary, .button-secondary {
+  padding: 1rem 3rem;
+  border-radius: 4px;
+  -webkit-box-shadow: 0 0.05em 0.25em 0 rgba(0, 0, 0, 0.5);
+          box-shadow: 0 0.05em 0.25em 0 rgba(0, 0, 0, 0.5);
+}
+
+a.button {
+  color: white;
+  text-decoration: none;
+}
+
+.button-primary {
+  background-color: #6abfc3;
+}
+
+.button-secondary {
+  background-color: #ce973e;
+}
+
+body {
+  font-size: 18px;
+  font-family: sans-serif;
+}
+
+header {
+  height: 94px;
+}
+
+li a {
+  color: rgba(74, 74, 74, 0.7);
+}
+
+.how-it-works img {
+  height: 165px;
+}
+
+.white-background {
+  background-color: white;
+}
+
+.language-icon-padding-margin {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  padding: 18px;
+  margin: 18px;
+}
+
+.box-shadow {
+  -webkit-box-shadow: 0 7px 16px 0 rgba(0, 0, 0, 0.05);
+          box-shadow: 0 7px 16px 0 rgba(0, 0, 0, 0.05);
+}
+
+.background-gradient {
+  background-image: -webkit-gradient(linear, left top, left bottom, from(#fff), to(#eee));
+  background-image: linear-gradient(to bottom, #fff, #eee);
+}
+
+.stories-photo {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+}
+
+footer img {
+  height: 60px;
+}
+
+.primary-menu {
+  display: none;
+}
+
+@media (min-width: 1124px) {
+  .primary-menu {
+    display: block;
+  }
+}
+
+.hamburger-menu {
+  display: block;
+}
+
+@media (min-width: 1124px) {
```

Comment:
@media (min-width: 1124px) is defined in multiple places I am not sure why that is, please take a good look at the code.